.gitlab-ci.yml

```
@@ -859,6 +859,27 @@ cudamemcheck:
     - cuda
     - gpu
 
+cudainitcheck:
+  stage: QoS_tools
+  image: localhost:5000/gko-cuda101-gnu8-llvm10-intel2019
+  before_script: *default_before_script
+  script:
+    - ctest -V -S cmake/CTestScript.cmake -DCTEST_BUILD_CONFIGURATION=RelWithDebInfo
+      -DCTEST_MEMORYCHECK_TYPE=CudaMemcheck "-DCTEST_MEMORYCHECK_COMMAND_OPTIONS=--tool;initcheck"
```

Sorry, messed this up again, it should be
```suggestion
      -DCTEST_MEMORYCHECK_TYPE=CudaMemcheck "-DCTEST_MEMORYCHECK_COMMAND_OPTIONS=--tool initcheck"
```